nginx.conf

```diff
@@ -0,0 +1,39 @@
+worker_processes 1;
+
+events {
+  worker_connections 1024;
+}
+
+error_log /dev/stderr;
+
+http {
+
+  server {
+    listen 80;
+    access_log /dev/stdout;
+
+    root /var/www/html;
+
+    location /test/ {
+        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+        proxy_set_header Connection "upgrade";
+        proxy_set_header Host $host;
+
+	    proxy_pass http://host.docker.internal:1880/dashboard/;
+
+        proxy_http_version 1.1;
+        proxy_set_header Upgrade $http_upgrade;
+    }
+
+    location /foo/bar/ {
+        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+        proxy_set_header Connection "upgrade";
+        proxy_set_header Host $host;
+
+	    proxy_pass http://host.docker.internal:1880/dashboard/;
+
+        proxy_http_version 1.1;
+        proxy_set_header Upgrade $http_upgrade;
+    }
+  }	
+}
```

Comment:
Did we mean to include an nginx conf in a clarification about icons?

Reply:
woops